app/views/layouts/megatron/application.html.slim

```diff
@@ -8,10 +8,12 @@ html lang="en"
     meta name="viewport" content="width=device-width,initial-scale=1"
     meta charset="utf-8"
     meta http-equiv="X-UA-Compatible" content="IE=edge"
+    meta name="bugsnag" data-api-key="#{ENV['BUGSNAG_API_KEY']}" data-release-stage="#{Rails.env}"
```

Comment:
I think `BUGSNAG_API_KEY` here should probably be `BUGSNAG_JS_API_KEY`, because the Rails side of the app will have a different key, and it's good to keep them straight.

Also, I believe `Rails.env` will always be `"production"`, so this might not be super useful. @abrookins @bpeicher any ideas?